gapipy/cache.py

```diff
@@ -168,3 +177,30 @@ def info(self):
 
     def is_cached(self, key):
         return self._client.exists(self.key_prefix + key)
+
+
+if django_settings:
+    class DjangoCache(BaseCache):
+        def __init__(self, *args, **kwargs):
+            assert django_settings.CACHES is not None, "No CACHES found in django settings!"
+            assert django_settings.CACHES.get('gapi') is not None, "No 'gapi' entry found in settings.CACHES!"
+            self.client = django_cache['gapi']
```

Comment:
If the import had succeeded, then `django_cache` is `from django.core.cache import cache`.

When I do that import in a local Django 1.11.29 install I get back an instance of `DefaultCacheProxy` which doesn't implement `__getitem__` and wouldn't allow us to reach an and grab a "gapi" cache -- I think it only ever proxies to the one named "default".

If instead we import the (note: plural "caches") `from django.core.cache import caches` then we can `__getitem__` to grab specific cache-backend named "gapi" or somesuch.

(Aside: I actually only went digging around here because I was curious about your comment on delete, actually, heh. I was curious about the `del cache[key]` API but I couldn't find any sign of it, maybe got dropped.)

Reply:
I should have checked more than one version of Django 🤦 
I was only working off the latest version of the source...

Reply:
Oh, funny, I did test on 1.11.29 and it was working fine 😕

Reply:
This also works 🤷